### ISSUE TITLE
Delegated events don't work for middle clicks

### ISSUE THREAD

**Issue by Getfree** [jquery/jquery#2954]
In this example, left-clicking the A element triggers both the normal listener and the delegated listener.
But if you middle-click, the delegated listener is not triggered.

Tested in Google Chrome and IE9.

```html
<a>click me</a>
```
```javascript
$('a').click(function(){ alert('normal') })
$('body').on('click','a',function(){ alert('delegated') })
```
[JSFiddle here](https://jsfiddle.net/myohedrc/3/)

**Comment by alecpl** [comment]
https://bugs.jquery.com/ticket/11485

**Comment by markelog** [comment]
Yep, it's a duplicate

**Comment by dwelle** [comment]
Wasn't this supposed to have been fixed in `1.12` via https://github.com/jquery/jquery/issues/2410?

/cc @dmethvin
